### PR TITLE
Displaying User Enrollments on Dashboard

### PR DESCRIPTION
**Pull request by clintonb**
User with enrollments:
user-enrollments

User without enrollments:
user-no-enrollments

@rlucioni @jimabramson A bit of playing around.

### CODE REVIEW

**Comment by jimabramson** [comment]
@clintonb I think we should handle an enrollment api timeout (or other error) gracefully, displaying something like "unable to retrieve enrollment information" instead of the table when there's a problem.

**Comment by clintonb** [comment]
@jimabramson Updated. I'll add tests for this later (before merging). It isn't in scope for this sprint, so I'm not going to rush it.

**Comment by clintonb** [comment]
@jimabramson I went ahead and wrote tests and created a ticket.

**Comment by clintonb** [comment]
XCOM-347

**Comment by rlucioni** [comment]
Nice. :+1:

File: ecommerce/extensions/dashboard/users/tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+from ecommerce.extensions.dashboard.users.views import UserDetailView
+from ecommerce.tests.mixins import UserMixin
+
+LOGGER_NAME = 'ecommerce.extensions.dashboard.users.views'
+
+
+class UserDetailViewTests(DashboardViewTestMixin, UserMixin, TestCase):
+    def setUp(self):
+        super(UserDetailViewTests, self).setUp()
+        self.switch, _created = Switch.objects.get_or_create(name='user_enrollments_on_dashboard', active=True)
```
Comment (jimabramson) [inline ecommerce/extensions/dashboard/users/tests/test_views.py]: nitpick: is it more idiomatic to use `__` for the unused var?

**Comment by jimabramson** [comment]
@clintonb nitpick is optional. :+1:   thanks for adding the failure handling for the enrollment call.

File: ecommerce/extensions/dashboard/users/tests/test_views.py (hunk 1 above)
Comment (clintonb) [inline ecommerce/extensions/dashboard/users/tests/test_views.py]: Updated.

**Pull request merged**